src/simian/auth/tlslite_bridge.py

```diff
@@ -30,7 +30,7 @@
   # we don't use this api, so safely stub out fcntl for appengine
   if (os.environ.get('SERVER_SOFTWARE', '').startswith('Google App Engine') or
       os.environ.get('SERVER_SOFTWARE', '').startswith('Development')):
-    logging.warning('stub out fcntl')
+    logging.info('stub out fcntl')
```

Comment:
In the AppEngine console logs - I like to monitor at warning level...
Reverted for now.